pkg/ingester/client/custom.go

```diff
@@ -65,3 +68,14 @@ func ChunkFromMeta(meta chunks.Meta) (Chunk, error) {
 func DefaultMetricsMetadataRequest() *MetricsMetadataRequest {
 	return &MetricsMetadataRequest{Limit: -1, LimitPerMetric: -1, Metric: ""}
 }
+
+// CloneUnsafe converts all of c's unsafe references to safe copies.
```

Comment:
Having "unsafe" in the name makes me think this is doing something with the `unsafe` module. Logic looks good though :+1:

Reply:
I suppose I should rename it to something less ominous sounding in that case. Would `CloneRefs` work?

Reply:
Pushed a revision renaming to `CloneRefs`, let me know what you think.